packages/ai/tests/lazy-tool-manager.test.ts

```
@@ -1,8 +1,9 @@
 import { describe, it, expect } from 'vitest'
 import type { Tool } from '../src/types'
-import { LazyToolManager } from '../src/activities/chat/tools/lazy-tool-manager'
-
-const DISCOVERY_TOOL_NAME = '__lazy__tool__discovery__'
+import {
+  DISCOVERY_TOOL_NAME,
+  LazyToolManager,
+} from '../src/activities/chat/tools/lazy-tool-manager'
```

ESLint

[error] `../src/activities/chat/tools/lazy-tool-manager` import should occur before type import of `../src/types`

(import/order)